parts/k8s/addons/kube-proxy.yaml

```diff
@@ -79,7 +79,7 @@ spec:
       containers:
       - command:
         - kube-proxy
-        - --config=/var/lib/kube-proxy/config.yaml
```

Comment:
Note: the above config reference points to a file that aks-engine doesn't pave to disk, so I'm just replacing it altogether